src/Events/Handlers/StatisticsSenderHandler.php

```diff
@@ -28,12 +29,21 @@ public function __construct(SendEventService $sendEventService)
      */
     public function handle(TaskContract $taskContract): bool
     {
-        if (!($taskContract instanceof SendEventTask)) {
+        if (
+            !($taskContract instanceof SendSummarizableEventTask) &&
```

Comment:
Please, create a common task for such statistics events. Because next time we should add here the next one and it really awful

Reply:
With the interface for eventDTO it can probably be easier to have the same class for this kind of task.